### DISCUSSION TITLE
Call unloader earlier

### DISCUSSION THREAD

**Discussion by ole22143** [Courseplay/Courseplay_FS22#3283] [Ideas]
Would it be an option to lower Combine call unloader limit i allways change in VehicleSettingsSetup.xml
		<Setting classType="AIParameterSettingList" name="callUnloaderPercent" min="60" max="90" incremental="5" default ="80" unit="4" isExpertModeOnly="false"/>
to 

		<Setting classType="AIParameterSettingList" name="callUnloaderPercent" min="30" max="90" incremental="5" default ="80" unit="4" isExpertModeOnly="false"/>

**Comment by Tensuko** [comment]
It was too early in the morning....

Indeed thats the correct setting, but lowering isn't a good Idea and we will stick to our min 60%.

**Reply by ole22143** [reply]
ah okay i testet it like this and there wehre no issues maybe more load for the server but on big fields it was helping

**Reply by Tensuko** [reply]
Yeah, but... I prefer the stable solution. :)

**Comment by rohne83** [comment]
Good to know. I sometimes missed the opportunity to lower it at least to 50%.
Use cases filling near-full unloader with a bit of remaining fruit, or especially convoys of combines for unloading the leading one before it starts into the next land (more "clean" solutions here would be either a button to manually initiate a call, or CP recognizes such situation automatically also for convoys and initiates the call at row-end itself).
